### DISCUSSION TITLE
Is there a `visible` or `enabled` prop?

### DISCUSSION THREAD

**Discussion by matthew-dean** [atomiks/tippyjs#1170] [General]
Use case - I'm using a Vue component which sets up a Tippy instance. In my data, I have Tippy tooltips which may or may not have content, so a Tippy tooltip may or may not need to be created. However, because this is Vue, I cannot easily opt to not include the directive, and the directive creates Tippy. So, I'm wondering if there's a way to pass a property to Tippy that says "okay, in this instance, do not create / show a tooltip". I was hoping for something like "enabled" or "disabled" or "visible", but i don't see anything like that? Is there another method? Note, I see there is a `disable()` and `enable()` instance method, but I don't see how to set that state on the props themselves?

**Comment by f4ik-md** [comment]
nuh uh.
